components/governance/ProposalComments.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="proposal-comments">
+    <div class="proposals-list--header proposal-comments--header">
+      <b-field class="field-tabs">
+        <template v-for="item in tabs">
+          <b-radio-button :key="item.id" v-model="currentTab" :native-value="item.id" type="is-primary">
+            <span>{{ $t(item.label) }}</span>
+          </b-radio-button>
+        </template>
+      </b-field>
+    </div>
+
+    <ProposalCommentsSkeleton v-if="isFetchingComments" :size="comments.length ? 1 : 3" />
+    <ProposalComment
```

Comment:
should pagination be added?

Reply:
should there be a v-else for ProposalComment?

Reply:
> should pagination be added?
real pagination is bad option (because it needs to be sorted), virtual - need to discuss

> should there be a v-else for ProposalComment? 
so conceived